samples/AutomaticTraceIdInjection/Log4NetExample/Properties/launchSettings.json

```diff
@@ -1,17 +1,11 @@
 {
   "profiles": {
-    "LaunchWithServiceUnification": {
+    "Log4NetExample": {
       "commandName": "Project",
       "environmentVariables": {
         "DD_LOGS_INJECTION": "true",
-        "DD_ENV": "customer-sample",
-        "DD_VERSION": "1.0.0-beta"
-      }
-    },
-    "LaunchWithoutServiceUnification": {
-      "commandName": "Project",
-      "environmentVariables": {
-        "DD_LOGS_INJECTION": "true"
+        "DD_VERSION": "1.0.0",
+        "DD_ENV": "dev"
```

Comment:
All the docs say users should also set `DD_SERVICE` (even though it's not needed in most cases). Should we include it here to avoid confusion? Alternatively, we can fix the docs.

Reply:
Added `DD_SERVICE` to all applications in cb3316fb048505a7317834cc701a7fd80908bdc2